app/javascript/components/pages/FateNew.js

```diff
@@ -15,68 +14,127 @@ class FateNew extends Component{
         description: "",
         is_public: false,
       },
-      submitted: false
-    }
+      submitted: false,
+      urls: [],
+    };
   }
 
   handleChange = (e) => {
-        // destructuring form out of state
-        const { form } = this.state
-        form[e.target.name] = e.target.value
-        // setting state to the updated form
-        this.setState({ form: form })
-    }
-    handleSubmit = (e) => {
-        e.preventDefault()
-        this.props.createNewEvent(this.state.form)
-        this.setState({submitted: true})
-    }
+    // destructuring form out of state
+    const { form } = this.state;
+    form[e.target.name] = e.target.value;
+    // setting state to the updated form
+    this.setState({ form: form });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.props.createNewEvent(this.state.form);
+    this.setState({ submitted: true });
+  };
 
-    render() {
+  fetchImages = async () => {
+    const response = await fetch(
+      `https://api.unsplash.com/search/photos/?client_id=1cb7RG1N3agHLuchuLs2GeqtKtdcyELRMj5HLHf8p48&query=${this.state.form.category}&per_page=3&orientation`
+    );
+    const test = await response.json();
```

Comment:
This name should be updated to reflect what it is now that we're not testing anymore. like `responseJson` maybe?

Reply:
Fixed to say response, i don't know if thats okay but i feel like that explains it okay.

Reply:
I think response was already defined? no?

Reply:
instead of the const `test` it should be like, `responseJson` it cant be response cause we're already using that var.

Reply:
yeah i completely glazed over it my bad its been fixed.